videoConf/views.py

```
@@ -0,0 +1,22 @@
+from django.views.decorators.http import require_http_methods
+from django.conf import settings
+
+
+# Create your views here.
+@require_http_methods(["GET"])
+@login_required
+def meeting(request: HttpRequest) -> HttpResponse:
+    context = { 
+        'room_name': 'life is beautiful',
+        'user_name': 'user_name',
+        'JITSI_MAGIC_COOKIE': settings.JITSI_MAGIC_COOKIE
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)